setup.cfg

```diff
@@ -0,0 +1,43 @@
+[flake8]
+max-line-length = 120
+exclude =
+    .git,
+    __pycache__,
+    **/migrations/,
+    build/,
+    .tox/,
+
+[isort]
+line_length = 79
```

Comment:
we should change this 88 align with black?

Reply:
Yes